daemon/daemon_unix.go

```diff
@@ -925,6 +928,47 @@ func driverOptions(config *config.Config) nwconfig.Option {
 	})
 }
 
+type defBrOptsV4 struct {
+	cfg config.BridgeConfig
+}
+
+func (o defBrOptsV4) nlFamily() int {
+	return netlink.FAMILY_V4
+}
+func (o defBrOptsV4) fixedCIDR() (fCIDR, optName string) {
+	return o.cfg.FixedCIDR, "fixed-cidr"
+}
+func (o defBrOptsV4) bip() (bip, optName string) {
+	return o.cfg.IP, "bip"
+}
+func (o defBrOptsV4) defGw() (gw net.IP, optName string) {
+	return o.cfg.DefaultGatewayIPv4, "DefaultGatewayIPv4"
```

Comment:
Is `DefaultGatewayIPV4` the name that will be used in the `daemon.json` ? I noticed that all other options we use hyphenated names, e.g. https://github.com/moby/moby/blob/48e43eb8601e5edaec3b4b88926413ef786e2913/daemon/config/config_linux.go#L62-L63


Also curious if it would make sense to make these options in the `internal/opts` package (but haven't looked too closely)